### ISSUE TITLE
Homepage Waitlist

### ISSUE THREAD

**Issue by kyle-hauber** [CodeForPhilly/prevention-point#86]
**User Story:**
As a front desk employee (FDE), I need to be able to view the current waitlist from the homepage, so that I can view and manage which participants should be seen next.

**Acceptance Criteria:**
- [x] The homepage needs to have a way for the FDE to toggle between the different programs.  This list will come from the back end.
- [x] For each program's waitlist, the following information should appear:
    - [x] Last Name
    - [x] UID
    - [x] Check-in Time
    - [x] Status
    - [x] Service
    - [x] "Seen" column - clicking this doesn't need to do anything at this stage.  In the future, however, this will be used to remove someone from the waitlist
    - [x]  "Notes" column - this column needs to exist but doesn't need to be functional at this state in order for this issue to be considered "done".  This functionality will be added via a separate issue: #123 
**Out of Scope:**
adding the stop button at the bottom of the page.  This can be implemented when the other time management features are complete.

**Images:**
https://www.figma.com/file/IikI37AjINeoyMxkL0emh4Er/Prevention-Point-V2?node-id=2%3A18

**Comment by kyle-hauber** [comment]
@AilinLuca or @cfazio - I've added a link to the HP waitlist to this ticket that @grahamalama is working on.  We need to make a few tweaks based on conversations today:
- [ ] Remove "seen" columns
- [ ] creates status options in drop down component with the following options:
    - [ ] waiting
    - [ ] seen
    - [ ] Left
    - [ ] Stepped Out

@grahamalama - you should be able to just reference the link in the description now.  The tweaks above will show up on that same link once finished (sorry for the mid-story change!)

**Comment by kyle-hauber** [comment]
@bkarlovitz , @grahamalama - Are you both working on this?  Is there is there another issue/PR that can be linked to this?  Just trying to get a handle on who's doing what and current status.  Let me know if there's anything I can help with or QA!

**Comment by kyle-hauber** [comment]
Blocked by #127

**Comment by jkeary** [comment]
Service issue is still a blocker and then writing tests and then it will be ready for a pr

**Comment by kyle-hauber** [comment]
Pending a response on #132 (which blocks #127), this may be ready to move forward again (i.e. unblocked)
